### ISSUE TITLE
Subcategories are not displayed consistently across all elements in the elements tree

### ISSUE THREAD

**Issue by modxbot** [modxbot/migrate#2370]
jrochman created Redmine issue ID 2370

Go to the elements tree. Create a category. Create another category. Drag the second category into the first. The other elements will not reflect the change. Visual representation. Templates
-Category
--Second Category
Template Variables
-Category
-Second Category
Chunks
-Category
-Second Category
Snippets
-Category
-Second Category
Plugins
-Category
-Second Category

**Comment by opengeek** [comment]
opengeek submitted:

Changed resources to elements in this ticket's summary and description to avoid unnecessary semantic confusion. In 0.9.7 forward, resources = documents or pages, and what used to be called resources in previous releases are now officially known as elements.

**Comment by splittingred** [comment]
splittingred submitted:

This is because there are no subcategories in 0.9.7. This may or may not make it into 097 development.

**Comment by modxbot** [comment]
jrochman submitted:

Then the system should disallow the nesting of categories by a dragging operation, thus giving the appearance that it allows the creation of subcategories.

**Comment by mkschell** [comment]
netprophet submitted:

Committed r.3819
In Element tree: user can no longer nest Category folders. drag-indicator icon is now more accurate about where you can and can not drop things
Tested FF2.0, IE6, Opera 9

**Comment by muzikman** [comment]
muzikman submitted:

I think that drag and drop of the category hierarchy is a great feature to have. I ran MODx Revolution 2.0.0-beta-4
Release Date: October 10th, 2009 and the categories where not nesting underneath the their parent counterparts whether I dragged the folder or right clicked and selected "Add New Resource" I think this should be fixed and implemented. The menu tree is an AJAX module if I'm not mistaken? If so, then it should act accordingly. Just my two cents.
